### PR TITLE
DE30375: Remove use-saved-searches attribute in favour of checking th…

### PR DESCRIPTION
**Pull request by njostonehouse**
…e tab collection's size.

### CODE REVIEW

File: src/d2l-my-courses-content/d2l-my-courses-behavior.html [hunk 1]
```diff
@@ -77,36 +74,40 @@
...
+					var promotedSearchesEntity = values[0];
+					var userSettingsEntity = values[1];
+
+					if (!promotedSearchesEntity || !promotedSearchesEntity.properties) {
+						return;
+					}
+
+					this._tabSearchType = promotedSearchesEntity.properties.UserEnrollmentsSearchType;
+
+					if(promotedSearchesEntity.actions.length < 2 || !promotedSearchesEntity.actions) {
```
Comment (ryantmer) [inline src/d2l-my-courses-content/d2l-my-courses-behavior.html]: Ah JS, where the order of `||` matters!

(The order of the conditions here is backwards 😄)

Comment (njostonehouse) [reply src/d2l-my-courses-content/d2l-my-courses-behavior.html]: Actually I think that condition shouldn't be here yet. Working ahead of myself.

Comment (ryantmer) [reply src/d2l-my-courses-content/d2l-my-courses-behavior.html]: Oh true - this is only needed for the "don't show tabs when only one tab" situation!

**Comment by njostonehouse** [comment]
Tests failing all over the place. Need to reconsider how my refactors work.

**Pull request closed without merging**